### DISCUSSION TITLE
Real-time value of activePower on home assistant

### DISCUSSION THREAD

**Discussion by stefanofalasca** [Koenkk/zigbee2mqtt#14148] [General]
Hi,
I configured zigbee2mqtt on home assistant and I configured a few devices Legrand 067775/741811. This device has haElectricalMeasurement cluster with activePower attribute. If I try to read value activePower from developer console, data is read correctly. I configured a view on board of home assistant to show current activePower with relative entity id, but value of power on dashboard is updated only when I read manually value from developer console, so I am not able to see realtime value automatically.
Somebody can help me ?
Thanks

**Comment by Slayer-24** [comment]
I have exactly the same issue with three smart plugs (Xiaomi | SP-EUC01). Updating with dev console works fine. But I get an error when I try to add 'haElectricalMeasurement/activePower' in the Reporting tab of Z2M.

**Comment by Slayer-24** [comment]
I did replace my CC2531 with a Sonoff CC2653P (with latest FW). After re-join my devices everything works fine.
